CMakeLists.txt

```diff
@@ -35,11 +35,16 @@ option(CESIUM_TRACING_ENABLED "Whether to enable the Cesium performance tracing
 option(CESIUM_COVERAGE_ENABLED "Whether to enable code coverage" OFF)
 option(CESIUM_TESTS_ENABLED "Whether to enable tests" ON)
 option(CESIUM_GLM_STRICT_ENABLED "Whether to force strict GLM compile definitions." ON)
+option(CESIUM_DISABLE_DEFAULT_ELLIPSOID "Whether to disable the WGS84 default value for ellipsoid parameters across cesium-native." OFF)
 
 if (CESIUM_TRACING_ENABLED)
     add_compile_definitions(CESIUM_TRACING_ENABLED=1)
 endif()
 
+if(CESIUM_DISABLE_DEFAULT_ELLIPSOID)
+  add_compile_definitions(CESIUM_DISABLE_DEFAULT_ELLIPSOID=1)
+endif()
```

Comment:
It would be better to move this to `CesiumGeospatial/CMakeLists.txt` and write it as:

```
if(CESIUM_DISABLE_DEFAULT_ELLIPSOID)
  target_compile_definitions(CesiumGeospatial PUBLIC CESIUM_DISABLE_DEFAULT_ELLIPSOID=1)
endif()
```

`add_compile_definitions` is an old-school function with global effect and is best avoided.